src/mixins/date-mixin.ts

```diff
@@ -1,51 +1,81 @@
 import { Component, Vue } from 'vue-property-decorator'
+import { isDate } from 'lodash'
 
-//
-// This mixin contains only the methods needed for the shared components.
-//
+/**
+ * This is a local, minimal mixin containing only the methods needed for the shared
+ * components when they run within this project (ie, in Storybook).
+ *
+ * When imported into an app project, the shared components should use the mixins
+ * local to that project.
+ */
 @Component({})
```

Comment:
As per this comment, this mixin only contains the methods needed by components in this library.

(These methods are copied directly from business-filings-ui.)